stubs/uWSGI/uwsgi.pyi

```diff
@@ -0,0 +1,241 @@
+from _typeshed import HasFileno, OptExcInfo, ReadOnlyBuffer
+from _typeshed.wsgi import WSGIApplication
+from collections.abc import Callable
+from types import ModuleType
+from typing import Any, Protocol, overload
+from typing_extensions import Literal, Self, TypeAlias, final
+
+import uwsgidecorators
+
+_TrueOrNone: TypeAlias = Literal[True] | None
+
+class _RPCCallable(Protocol):
+    def __call__(self, *args: bytes) -> bytes | None: ...
+
+# FIXME: Technically we know the exact layout of _AppsDict and _WorkerDict
+#        but TypedDict does not support bytes keys, so for now we use type
+#        aliases to a more generic dict
+_AppsDict: TypeAlias = dict[bytes, int | bytes]
+_WorkerDict: TypeAlias = dict[bytes, int | bytes | tuple[_AppsDict, ...]]
```

Comment:
These will be a bit annoying to use, because you need to do type narrowing with `assert`:

```python
value = worker_dict[b"foo"]
assert isinstance(value, int)
# use the value
```

So I think this is the best we can do:

```suggestion
_WorkerDict: TypeAlias = dict[bytes, Any]
```

Reply:
Yeah I've debated with myself about whether or not I should use a union here. I'm fine with going this route for now. Once either `AnyOf` or `TypedDict`/`TypedMapping` with byte keys become a thing we can make it more specific.